src/Dapi/Types/CategorizedTransaction.cs

```diff
@@ -0,0 +1,36 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Dapi.Types {
+    public class CategorizedTransaction {
+        public float amount { get; }
+        public string date { get; }
+        public Dapi.Types.TransactionType type { get; }
+        public string description { get; }
+        public string details { get; }
+        public Currency currency { get; }
+        public float beforeAmount { get; }
+        public float afterAmount { get; }
+        public string reference { get; }
+        public string category { get; }
+
+        public CategorizedTransaction(float amount, string date, TransactionType type, string description, string details, Currency currency, float beforeAmount, float afterAmount, string reference, string category) {
+            this.amount = amount;
+            this.date = date;
+            this.type = type;
+            this.description = description;
+            this.details = details;
+            this.currency = currency;
+            this.beforeAmount = beforeAmount;
+            this.afterAmount = afterAmount;
+            this.reference = reference;
+            this.category = category;
+        }
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public enum TransactionType {
```

Comment:
Since this type will not be used anymore, you should delete it.

Reply:
eb69a3a20e545504c52b4ed348344b8395581328